src/time.cr

```diff
@@ -471,21 +471,51 @@ struct Time
     (year % 4 == 0 && year % 100 != 0) || (year % 400 == 0)
   end
 
-  def inspect(io : IO)
-    case
-    when utc?
-      to_s "%F %T UTC", io
-    else
-      if offset % 60 == 0
-        to_s "%F %T %:z", io
+  # Prints this `Time` to *io*.
+  #
+  # The local date-time is formatted as ISO 8601 date string `YYYY-MM-DD HH:mm:ss.nnnnnnnnn +ZZ:ZZ:ZZ`.
```

Comment:
that string isn't iso 8601 and the example doesn't include the zone name.

Reply:
Why isn't it ISO 8601?

Reply:
Zone name is mentioned in the next paragraph.

Reply:
no T, and there's a space between the nanoseconds and the +

Reply:
`T` designator can be omitted, but the additional white space is true. The question is: should it be removed?